spectral/src/functions/expectServersConfig.ts

```diff
@@ -1,6 +1,6 @@
-import { IFunctionPaths, IFunctionResult } from "@stoplight/spectral";
+import { IFunctionResult } from "@stoplight/spectral-core";
 
-export default (targetVal: any, opts: any, paths: IFunctionPaths): IFunctionResult[] => {
```

Comment:
Side note/Documentation. Servers rule only applies to control plane/management APIs. 
Instance APis would typically have their own servers supplied dynamically.